docs/plugins/contributors/fallback.json

```diff
@@ -0,0 +1,34 @@
+{
+  "login": "vepor",
+  "id": 7254437,
+  "node_id": "MDQ6VXNlcjcyNTQ0Mzc=",
+  "avatar_url": "https://avatars.githubusercontent.com/u/7254437?v=4",
+  "gravatar_id": "",
+  "url": "https://api.github.com/users/vepor",
+  "html_url": "https://github.com/vepor",
+  "followers_url": "https://api.github.com/users/vepor/followers",
+  "following_url": "https://api.github.com/users/vepor/following{/other_user}",
+  "gists_url": "https://api.github.com/users/vepor/gists{/gist_id}",
+  "starred_url": "https://api.github.com/users/vepor/starred{/owner}{/repo}",
+  "subscriptions_url": "https://api.github.com/users/vepor/subscriptions",
+  "organizations_url": "https://api.github.com/users/vepor/orgs",
+  "repos_url": "https://api.github.com/users/vepor/repos",
+  "events_url": "https://api.github.com/users/vepor/events{/privacy}",
+  "received_events_url": "https://api.github.com/users/vepor/received_events",
+  "type": "User",
+  "site_admin": "",
+  "name": "Luděk Vepřek",
+  "company": "Kiwi.com",
+  "blog": "",
+  "location": "Czech Republic",
+  "email": "",
+  "hireable": "",
+  "bio": "",
+  "twitter_username": "",
+  "public_repos": 5,
+  "public_gists": 1,
+  "followers": 9,
+  "following": 6,
+  "created_at": "2014-04-10T18:16:16Z",
+  "updated_at": "2021-01-27T10:58:04Z"
+}
```

Comment:
it's needed, you can't run gatsby website with that error, because it will fail on graphql query, which will try to use non-existing schema 🤔

Reply:
Dummy data is necessary if you aren't going to customize the schema. Otherwise, GraphQL won't be able to run the query. I don't think we need any fields that we aren't querying for, though.

But if you're going to go to the trouble of creating the dummy data, it could easily be replicated in a schema customization. Then if there's a null value/empty array, you can use the warning Alert. I think that kind of experience would be easier for developers.

Reply:
@CollierCZ you mean typing it explicitly how you suggested before? how we can achieve that with remote schema?

Reply:
yep, my fault, i tried to extend external graphql schema, so it did not work, but when i switched to rest api, i forgot to customize it and choosed that way with creating dummy node

Reply:
I'm still unsure why you can't define the schema directly, as we know what fields we're looking for. And I'm pretty sure [external schema get merged](https://www.gatsbyjs.com/docs/schema-generation/#6-merging-in-third-party-schemas) into the Gatsby schema. But if it works, it works.

Reply:
Did you check the last commit? It's explicitly defined, everything is ok, but it's not external schema anymore, since we don't use `graphql-api`, we use rest api now, that is why it works, i think, i did same before with external and it was not working.